app/models/to_do.rb

```diff
@@ -0,0 +1,3 @@
+class ToDo < ActiveRecord::Base
+
```

Comment:
I don't remember if the syllabus requires validations, but consider adding some! What if the todo has no title, for instance; is that valid data?